awx/main/tests/functional/test_migrations.py

```diff
@@ -73,18 +73,25 @@ def test_receptor_address(self, migrator):
     def test_migrate_DAB_RBAC(self, migrator):
         old_state = migrator.apply_initial_migration(('main', '0190_alter_inventorysource_source_and_more'))
         Organization = old_state.apps.get_model('main', 'Organization')
+        Team = old_state.apps.get_model('main', 'Team')
         User = old_state.apps.get_model('auth', 'User')
 
         org = Organization.objects.create(name='arbitrary-org', created=now(), modified=now())
         user = User.objects.create(username='random-user')
         org.read_role.members.add(user)
+        org.member_role.members.add(user)
+
+        team = Team.objects.create(name='arbitrary-team', organization=org, created=now(), modified=now())
+        team.member_role.members.add(user)
 
         new_state = migrator.apply_tested_migration(
             ('main', '0192_custom_roles'),
         )
 
         RoleUserAssignment = new_state.apps.get_model('dab_rbac', 'RoleUserAssignment')
         assert RoleUserAssignment.objects.filter(user=user.id, object_id=org.id).exists()
+        assert RoleUserAssignment.objects.filter(user=user.id, role_definition__name='Controller Organization Member', object_id=org.id).exists()
+        assert RoleUserAssignment.objects.filter(user=user.id, role_definition__name='Controller Team Member', object_id=team.id).exists()
```

Comment:
:+1:  alright, looks good here.

Reply:
Did that not require any changes to the data migration? I guess maybe it imported some of the translation methods from the models? I just don't see how this would have gotten fixed.